src/components/detailsPage/StatsCard/StatsCard.test.tsx

```diff
@@ -36,5 +36,6 @@ describe("StatsCard component", () => {
 
     expect(getByText("TYPE")).toBeInTheDocument();
     expect(getByText("Electric")).toBeInTheDocument();
+    expect(getByText("GAME INDICES")).toBeInTheDocument();
```

Comment:
You will also want to check and make sure the actual Game Indices are rendering. Like what you see above with the Type

Reply:
Elaborate on "what you see above with type"

Reply:
There is the TYPE section, and in the TYPE section there is the formatted text "Electric"
Similarly, there is the GAME INDICES section, and in the GAME INDICES section there is formatted text that you want to be checking for here

Reply:
I'm not sure what exactly you mean. How do I ensure that I'm checking for formatted text in this context, aside from what I've added here on line 39?

Reply:
You would add an additional statement checking to make sure the actual content is rendering. This is just checking for the heading

Reply:
I think I have this fixed now